src/data/nat/factorial/big_operators.lean

```diff
@@ -0,0 +1,35 @@
+/-
+Copyright (c) 2022 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kyle Miller, Pim Otte
+-/
+import data.nat.factorial.basic
+import algebra.big_operators.order
+
+/-!
+# Factorial with big operators
+
+This file contains some lemma's on factorials in combination with big operators
```

Comment:
```suggestion
This file contains some lemmas on factorials in combination with big operators.
```
Can you add a sentence explaining why a different file is needed please? Thanks!